testnet/stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -11379,3 +11380,646 @@ fn rbf_on_config_change() {
+    let timer = Instant::now();
+
+    // Fill the mempool with the transfers
+    for _ in 0..25 {
+        for (sender_sk, nonce) in senders.iter_mut() {
+            let sender_addr = tests::to_addr(sender_sk);
+            let fee = set_fee();
+            assert!(fee >= 180 && fee <= 2000);
+            let transfer_tx = make_stacks_transfer(
+                sender_sk,
+                *nonce,
+                fee,
+                naka_conf.burnchain.chain_id,
+                &recipient,
+                1,
+            );
+            insert_tx_in_mempool(
+                &db_tx,
+                transfer_tx,
+                &sender_addr,
+                *nonce,
+                fee,
+                &tip.consensus_hash,
+                &tip.canonical_stacks_tip_hash,
+                tip.stacks_block_height,
+            );
+            *nonce += 1;
+        }
+    }
+    db_tx.commit().unwrap();
+
+    info!("Sending transfers took {:?}", timer.elapsed());
+
+    info!("Mining transfers");
+    let proposed_blocks_before = test_observer::get_mined_nakamoto_blocks().len();
+
+    // Unpause block mining
+    TEST_MINE_STALL.set(false);
+
+    // Wait for the first block to be proposed.
+    wait_for(30, || {
+        let proposed_blocks = test_observer::get_mined_nakamoto_blocks().len();
+        Ok(proposed_blocks > proposed_blocks_before)
+    })
+    .expect("Timed out waiting for first block to be mined");
+
+    let blocks = test_observer::get_mined_nakamoto_blocks();
+    let last_block = blocks.last().unwrap();
+    info!(
+        "First block contains {} transactions",
+        last_block.tx_events.len()
+    );
+    if strategy == MemPoolWalkStrategy::NextNonceWithHighestFeeRate {
+        assert!(last_block.tx_events.len() > 5000);
+    }
+
+    // Wait for the transfers to all be mined
+    wait_for(7200, || {
+        for (sender_sk, nonce) in senders.iter() {
+            let sender_addr = tests::to_addr(sender_sk);
+            let account = get_account(&http_origin, &sender_addr);
+            if account.nonce < *nonce {
+                return Ok(false);
+            }
+        }
+        Ok(true)
+    })
+    .expect("Timed out waiting for transfers to be mined");
+
+    info!("Mining transfers took {:?}", timer.elapsed());
+
+    coord_channel
+        .lock()
+        .expect("Mutex poisoned")
+        .stop_chains_coordinator();
+    run_loop_stopper.store(false, Ordering::SeqCst);
+
+    run_loop_thread.join().unwrap();
+}
+
+#[test]
+#[ignore]
+fn large_mempool_original_constant_fee() {
+    large_mempool_base(MemPoolWalkStrategy::GlobalFeeRate, || 180);
+}
+
+#[test]
+#[ignore]
+fn large_mempool_original_random_fee() {
+    large_mempool_base(MemPoolWalkStrategy::GlobalFeeRate, || {
+        thread_rng().gen_range(180..2000)
+    });
+}
+
+#[test]
+#[ignore]
+fn large_mempool_next_constant_fee() {
+    large_mempool_base(MemPoolWalkStrategy::NextNonceWithHighestFeeRate, || 180);
+}
```

Comment:
Is there any functional difference between this and the implementation in the signer tests? If not, should we just remove this, given that these are ignored in CI anyways?